great_expectations/checkpoint/actions.py

```diff
@@ -130,10 +187,13 @@ class (for open source Great Expectations) or the GeCloudIdentifier (from Great
 
 
 class DataDocsAction(ValidationAction):
-    def __init__(self) -> None:
+    _build_data_docs: Callable = PrivateAttr()
+    _get_docs_sites_urls: Callable = PrivateAttr()
+
+    def __init__(self, **data: Any) -> None:
```

Comment:
In order to set the private attr, we override `BaseModel.__init__`. This will be any params from any of the subclasses so it needs to be a `**kwargs`.

I'd rather we not do this but I'm not sure how to set private attrs without some kind of post init behavior. You can't set with a root validator and we don't have an explicit post init hook like we would with dataclasses.